### PR TITLE
adding missing changelogs

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

add missing changelogs... for 1.1.0RC2

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests? 
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:  

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified; _Bugfixes should be including in bug-fix release milestones (m.f.X) and features should be included in (m.X.b) releases._
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,6 +73,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
+- Enabled `self.log` in most functions ([#4969](https://github.com/PyTorchLightning/pytorch-lightning/pull/4969))
+
+
+- Added changeable extension variable for model checkpoints ([#4977](https://github.com/PyTorchLightning/pytorch-lightning/pull/4977))
+
+
+- Added Commet `experiment_key` argument to online mode example ([#4997](https://github.com/PyTorchLightning/pytorch-lightning/pull/4997))
+
+
```
Comment (rohitgr7) [inline CHANGELOG.md:84]:
```suggestion
```
this was just a doc update. The argument was already added some timebefore.

File: CHANGELOG.md [hunk 2]
```diff
@@ -79,6 +73,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added `Pytorch Geometric` integration example with Lightning ([#4568](https://github.com/PyTorchLightning/pytorch-lightning/pull/4568))
 
 
+- Enabled `self.log` in most functions ([#4969](https://github.com/PyTorchLightning/pytorch-lightning/pull/4969))
+
+
+- Added changeable extension variable for model checkpoints ([#4977](https://github.com/PyTorchLightning/pytorch-lightning/pull/4977))
```
Comment (rohitgr7) [inline CHANGELOG.md:79]:
```suggestion
- Added changeable extension variable for `ModelCheckpoint` ([#4977](https://github.com/PyTorchLightning/pytorch-lightning/pull/4977))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -117,12 +126,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- Fixed `LoggerConnector` to have logged metrics on root device in DP ([#4138](https://github.com/PyTorchLightning/pytorch-lightning/pull/4138))
+
+
+- Fixed `PYTHONPATH` for ddp test model ([#4528](https://github.com/PyTorchLightning/pytorch-lightning/pull/4528))
+
+
+- Fixed allowing logger to support indexing ([#4595](https://github.com/PyTorchLightning/pytorch-lightning/pull/4595))
+
+
+- Fixed DDP and `manual_optimisation` ([#4976](https://github.com/PyTorchLightning/pytorch-lightning/pull/4976))
```
Comment (rohitgr7) [inline CHANGELOG.md:145]:
```suggestion
- Fixed DDP and manual_optimization ([#4976](https://github.com/PyTorchLightning/pytorch-lightning/pull/4976))
```

**Review by tchaton — APPROVED** [review]
LGTM !

File: CHANGELOG.md [hunk 4]
```diff
@@ -117,12 +123,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
 - Fixed `LoggerConnector` to have logged metrics on root device in DP ([#4138](https://github.com/PyTorchLightning/pytorch-lightning/pull/4138))
 
 
 - Auto convert tensors to contiguous format when `gather_all` ([#4907](https://github.com/PyTorchLightning/pytorch-lightning/pull/4907))
 
 
+- Fixed `LoggerConnector` to have logged metrics on root device in DP ([#4138](https://github.com/PyTorchLightning/pytorch-lightning/pull/4138))
+
+
```
Comment (awaelchli) [inline CHANGELOG.md:135]:
```suggestion
```
This is duplicated

**Review by awaelchli — APPROVED** [review]

**Review by s-rog — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Pull request merged**